### PR TITLE
adjust admission plugins order in kubeadm

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What this PR does / why we need it**:

follow up of #58040 , adjust admission plugins order in kubeadm

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
ping @luxas

**Comment by hzxuzhonghu** [comment]
@fejta The robot is crazy.

File: cmd/kubeadm/app/phases/controlplane/manifests.go [hunk 1]
```diff
@@ -42,9 +42,9 @@ import (
 const (
 	DefaultCloudConfigPath = "/etc/kubernetes/cloud-config"
 
-	defaultV18AdmissionControl    = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
-	deprecatedV19AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
-	defaultV19AdmissionControl    = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
+	defaultV18AdmissionControl    = "NamespaceLifecycle,Initializers,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
```
Comment (timothysc) [inline cmd/kubeadm/app/phases/controlplane/manifests.go]: I believe Initializers has been removed.

**Review by timothysc — COMMENTED** [review]
I don't this this matters anymore, if not please close.

**Comment by hzxuzhonghu** [comment]
close, since  #58123  already make `admission-control` orders not matters

**Pull request closed without merging**